docs/source/sysadmin_guide/off_chain_settings.rst

```diff
@@ -0,0 +1,300 @@
+****************************************************
+Changing Off-chain Settings with Configuration Files
+****************************************************
+
+Each Sawtooth component, such as the validator or the REST API, can have an
+optional configuration file that controls the component's behavior. By default,
+Sawtooth does not install any configuration files.  However, Sawtooth provides
+example configuration files that can be customized for your system.
+
+This procedure explains how to use Sawtooth configuration files to change the
+following Sawtooth settings:
+
+* Host and port values (bind endpoints) for the validator, REST API, and
+  consensus proxy
+
+* Peering type and peer nodes on the network
+
+* Network keys for secured communication between nodes (optional)
+
+* Scheduler type (optional)
+
+It also explains how to configure a non-default REST API URL for the Sawtooth
+commands.
+
+See :doc:`configuring_sawtooth` for detailed information on all the settings in
+each configuration file.
+
+.. note::
+
+   This procedure assumes that the configuration directory is
+   ``/etc/sawtooth/``. If your system uses a different location, change this
+   path in the commands below. For more information, see
+   :doc:`configuring_sawtooth/path_configuration_file`.
+
+.. _change-nw-validator-label:
+
+Configure the Validator
+=======================
+
+The following steps configure the validator's networking information so that the
+validator advertises itself properly and knows where to search for peers.
+Additional steps specify the peers for this node, change the scheduler type
+(optional), and create a network key.
+
+#. Create the validator configuration file by copying the example file.
+
+   .. code-block:: console
+
+      $ sudo cp /etc/sawtooth/validator.toml.example /etc/sawtooth/validator.toml
+
+#. Use ``sudo`` to edit this file.
+
+   .. code-block:: console
+
+      $ sudo vi /etc/sawtooth/validator.toml
+
+
+#. Change the network settings for the validator.
+
+   a. Locate the ``endpoint`` setting, which specifies validator's external URL.
+
+      Replace the default interface and port (``127.0.0.1:8800``) with the
+      values for your node. You can use either the NAT values or the publicly
+      addressable IP address and port.
+
+      .. code-block:: ini
+
+         endpoint = "tcp://{external_interface}:{port}"
+
+   #. Locate the ``bind`` settings. If necessary, change these values for your
+      system. The default values are:
+
+        .. code-block:: ini
+
+           bind = [
+             "network:tcp://127.0.0.1:8800",
+             "component:tcp://127.0.0.1:4004",
+             "consensus:tcp://127.0.0.1:5050"
+           ]
+
+      - ``network`` specifies where the validator listens for communication from
+        validators on other nodes
+
+      - ``component`` specifies where the validator listens for the REST API
+
+      - ``consensus`` specifies the settings for the consensus proxy
```

Comment:
I would change this to:
```
      - ``network`` specifies where the validator listens for communication from other validator nodes
      - ``component`` specifies where the validator listens for communication from this validator’s
         components, such as the REST API and transaction processors
      - ``consensus`` specifies where the validator listens for communication from consensus engines 
```
This is slightly more consistent with the wording here: https://sawtooth.hyperledger.org/docs/core/nightly/master/app_developers_guide/creating_sawtooth_network.html#prereqs-multi-ubuntu-label

And includes TPs in the component description and more accurately defines the consensus endpoint.